.github/workflows/testing.yml

```diff
@@ -56,7 +53,7 @@ jobs:
         if: matrix.dependencies == 'conda'
         run: |
           set -vxeo pipefail
-          conda install -y -c conda-forge doct jsonschema mock pymongo pytest pyyaml requests tornado ujson
+          conda install -y -c conda-forge doct jsonschema mock mongomock pymongo pytest pytest-tornado==0.8.1 pyyaml requests tornado ujson
```

Comment:
I think we can remove pytest-tornado

Reply:
was able to get rid of this in https://github.com/NSLS-II/conftrak/pull/7/commits/5df5ec3d9129cc73ae3f81534ed9080e77034821